docs/designers-developers/developers/tutorials/block-tutorial/writing-your-first-block-type.md

```diff
@@ -11,18 +11,19 @@ While the block's editor behaviors are implemented in JavaScript, you'll need to
 ```php
 <?php
 
```

Comment:
I've noticed that the plugin misses the header so it's shown in the plugins page:

```suggestion
/*
Plugin Name: Gutenberg examples 01
*/
```